src/screens/WebArticleScreen.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { WebView, View, StyleSheet } from 'react-native';
+
+export default class WebArticleScreen extends Component {
+  static navigationOptions = ({ navigation }) => {
+    const { url } = navigation.state.params;
+    return {
+      title: url,
+    };
+  };
+
+  constructor(props) {
+    super(props);
+    const { navigation } = this.props;
+
+    this.state = { url: navigation.state.params.url };
```

Comment:
do you need this in `state`, can it be accessed via `props` in render?
Also try to use `navigation.getParam()` instead of drilling through `navigation` object model